src/Adapter/PlatformServices.Shared/netstandard1.3/Resources/Resource.Designer.cs

```diff
@@ -40,7 +41,7 @@ internal class Resource {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
                     global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Resources.Reso" +
-                            "urce", typeof(Resource).Assembly);
+                            "urce", typeof(Resource).GetTypeInfo().Assembly);
```

Comment:
Resource.Designer file should not be edited manually. It should auto-generate

Reply:
Will remove from my committed files. But given the complications with resources file sharing b/w different projects, There would be some manual effort.
File will auto-generate and it uses full framework generator for this. To make it work for netcore you'll have to add a line GetTypeInfo() while getting the assembly details as the libraries have changes in netcore.